.github/workflows/ci.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - name: Run script tests
         run: yarn test:script:cov
       - name: Run integration tests
+        timeout-minutes: 5
```

Comment:
I believe this may have been a leftover of disabling the integration tests?

```suggestion
```

Reply:
I actually left it there on purpose, because when the integration tests didn't run as expected, it was just endlessly waiting for the headless browser to start. That caused the CI to run much longer than it should. I believe leaving the timeout in for now should be OK. What do you think?

Reply:
:+1: Sounds great!